### ISSUE TITLE
How do you do equivalent of Common?

### ISSUE THREAD

**Issue by SolsticeSpectrum** [p-sun/Shadertoy-to-LensStudio-Conversion-Guide#1]
I need some functions and constants defined for all buffers, do I just have  to redefine then in each buffer?

**Comment by p-sun** [comment]
Yes, you'd need to redefine them. There isn't a way to share code between Code Nodes at the moment.
